### PR TITLE
add some docs about deployments with teresa

### PR DESCRIPTION
**Pull request by drgarcia1986**
depends on #3

### CODE REVIEW

**Comment by lucianokadoya-luizalabs** [comment]
:+1:

**Review by alexguerra-luizalabs — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,141 @@
 # Hello Teresa
 A collection of `Hello World` ready to deploy on [teresa](https://github.com/luizalabs/teresa-api) PaaS.
+
+## Deploy
+
+Some information to up and run your application on Kubernetes with Teresa.
```
Comment (arnaldopereira) [inline README.md:6]: I'd write: `Some information to get your application up and running on Kubernetes with Teresa`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,141 @@
...
+
+### Port
+Don't use a fixed port to up your web application, instead,
```
Comment (arnaldopereira) [inline README.md:9]: ```Don't listen on a hardcoded port, but instead read the port from the environment variable `PORT`. For instance:```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,141 @@
...
+
+```go
+port := os.Getenv("PORT")
+if port == "" {
+    port = "5000"
+}
+http.ListenAndServe(fmt.Sprintf(":%s", port), nil)
+```
+
+The deploy process will set this environment variable
```
Comment (arnaldopereira) [inline README.md:20]: s/environment//

File: README.md [hunk 4]
```diff
@@ -1,2 +1,141 @@
...
+```
+
+Teresa follows the same principle.
+As an example, a Python application might have the following command on Procfile:
+
+    web: gunicorn -b 0.0.0.0:$PORT -w 3 --pythonpath src myapp.wsgi
+
+
+### Language detection
+When you make a deploy on Teresa you don't need to specify the language of your application,
```
Comment (arnaldopereira) [inline README.md:37]: `When you deploy an app using Teresa, you don't have to specify your application's language - it'll be automatically detected`

**Pull request merged**